azure_functions_worker/constants.py

```diff
@@ -43,11 +43,13 @@
 PYTHON_ENABLE_WORKER_EXTENSIONS_DEFAULT_39 = True
 PYTHON_EXTENSIONS_RELOAD_FUNCTIONS = "PYTHON_EXTENSIONS_RELOAD_FUNCTIONS"
 
+# new programming model default script file name
+SCRIPT_FILE_NAME = "SCRIPT_FILE_NAME"
```

Comment:
Is this going to be the AppSetting name?

Reply:
If yes, the please add PYTHON_FUNCTIONS or PYTHON_ prefix - helps make it explicit for Python only.

Reply:
Changed to PYTHON_SCRIPT_FILE_NAME. Final name might still need discussion, but PYTHON_ prefix will be included.